### PR TITLE
fix: multi tenant setup

### PR DESCRIPTION
**Pull request by debendraoli**

### CODE REVIEW

**Comment by debendraoli** [comment]
no idea why prosody check failed, anything to do with mine?

**Comment by saghul** [comment]
I don't think it was your fault, let me retry the job.

**Comment by debendraoli** [comment]
Probably something to do with CI cache?
looks like keyring issue, outdated perhaps.

**Comment by saghul** [comment]
Ping @damencho can you review? (The PR not the CI failure)

**Comment by damencho** [comment]
Looks good, just a small thing @debendraoli can you put spaces between string and `+`, thanks.

File: web/rootfs/defaults/system-config.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ if (subdir.startsWith('<!--')) {
 if (subdomain) {
     subdomain = subdomain.substr(0,subdomain.length-1).split('.').join('_').toLowerCase() + '.';
 }
-config.hosts.muc = '{{ $XMPP_MUC_DOMAIN_PREFIX }}.'+subdomain+'{{ $XMPP_DOMAIN }}';
+config.hosts.muc = '{{ $XMPP_MUC_DOMAIN_PREFIX }}. ' + subdomain + '{{ $XMPP_DOMAIN }}';
```
Comment (damencho) [inline web/rootfs/defaults/system-config.js:31]: This space will break the jid :)

Comment (damencho) [reply web/rootfs/defaults/system-config.js]: I fixed it

**Review by damencho — APPROVED** [review]

**Pull request merged**